tests/functional/scripts/pyi_helloworld.py

```diff
@@ -0,0 +1,11 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2015-2016, PyInstaller Development Team.
+#
+# Distributed under the terms of the GNU General Public License with exception
+# for distributing bootloader.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+
+
+print('Hello Python!')
```

Comment:
This is re adding a missing file needed for the osx_override_info_plist test which uses a spec file which requires this file.